### PR TITLE
Contribute French training script of Common Voice

### PR DESCRIPTION
**Pull request by wrz1999**
Contributed the French training script. The data set takes about 902 hours. Pandas and ffmpeg libraries need to be installed

### CODE REVIEW

**Review by robin1001 — CHANGES_REQUESTED** [review]
Great job. `tools` and `wenet` should be soft links, please refine it.

**Comment by robin1001** [comment]
And if the corpus is from common voice, please rename the directory to commonvoice/french or commonvoice/fr

**Comment by wrz1999** [comment]
> And if the corpus is from common voice, please rename the directory to commonvoice/french or commonvoice/fr

Ok I will fix it

File: examples/commonvoice/fr/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Performance Record
+# Should be installed ffmpeg , pandas !!!
```
Comment (robin1001) [inline examples/commonvoice/fr/README.md:2]: This line is just a note, not a header, right?

Comment (wrz1999) [reply examples/commonvoice/fr/README.md:2]: Yes

File: examples/commonvoice/fr/local/tmp.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```
Comment (robin1001) [inline examples/commonvoice/fr/local/tmp.py:1]: rename the file according to the function of the file.

File: examples/commonvoice/fr/run.sh [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+# Default 1 for single machine/node
+# NFS will be needed if you want run multi-machine training
+num_nodes=1
+# The rank of each node or machine, range from 0 to num_nodes -1
+# The first node/machine sets node_rank 0, the second one sets node_rank 1
+# the third one set node_rank 2, and so on. Default 0
+node_rank=0
+# data
+download_path=/root/autodl-tmp
+France_data=/root/autodl-tmp/cv-corpus-8.0-2022-01-19
```
Comment (robin1001) [inline examples/commonvoice/fr/run.sh:26]: rename it to french_data

File: examples/commonvoice/fr/local/prepare_data.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+done
+
+if [ ! -d ${src_path}/wavs ]; then
+    mkdir ${src_path}/wavs
+fi
+for x in train dev test; do
+    python3 ${local}/tmp.py  ${src_path} ${x} ${dir}/${x}
+done
+for x in train dev test; do
+    cat ${dir}/${x}/utt2spk | $local/utt2spk_to_spk2utt.pl > ${dir}/${x}/spk2utt || exit 1;
```
Comment (robin1001) [inline examples/commonvoice/fr/local/prepare_data.sh:32]: spk2utt/utt2spk is not required in wenet, you can just remove the line, and remove utt2spk_to_spk2utt.pl

**Review by robin1001 — APPROVED** [review]

File: examples/commonvoice/fr/run.sh (hunk 3 above)
Comment (egorsmkv) [reply examples/commonvoice/fr/run.sh]: Maybe it's better to use `/tmp` path instead of `/root` ?

**Pull request merged**